runtime/compiler/env/j9methodServer.hpp

```diff
@@ -305,6 +305,14 @@ class TR_ResolvedJ9JITaaSServerMethod : public TR_ResolvedJ9Method
    UnorderedMap<TR_ResolvedMethodKey, TR_ResolvedMethodCacheEntry> _resolvedMethodsCache;
    TR_IPMethodHashTableEntry *_iProfilerMethodEntry;
 
+   struct TR_IsUnresolvedString
+      {
+      TR_IsUnresolvedString(bool optForAOT, bool notOptForAOT): _optForAOT(optForAOT), _notOptForAOT(notOptForAOT) {}
+      bool _optForAOT;
+      bool _notOptForAOT;
```

Comment:
Can you change these to something like `_optimizeForAOTTrueResult` and `_optimizeForAOTFalseResult`? Having "optimizeForAOT" as part of the name helps readers relate it to the original parameter, find it when searching, etc.